src/Contrib/composer.json

```diff
@@ -15,7 +15,7 @@
         "ext-json": "*",
         "open-telemetry/api": "^1.0",
         "open-telemetry/context": "^1.0",
-        "open-telemetry/gen-otlp-protobuf": "^1.0",
+        "open-telemetry/gen-otlp-protobuf": "^1.1",
```

Comment:
Can we remove this `composer.json`? Doesn't seem to be used https://github.com/open-telemetry/opentelemetry-php/blob/main/.gitsplit.yml.

Reply:
I think we can - it was once used for https://packagist.org/packages/open-telemetry/sdk-contrib but that's now abandoned.